src/config.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+const herokuConfig = {
+  topic: process.env.KAFKA_LOGGING_TOPIC ? process.env.KAFKA_LOGGING_TOPIC : 'refocus-whitelist',
+  sslCert: process.env.KAFKA_CLIENT_CERT || '.ssl/client.crt',
+  sslKey: process.env.KAFKA_CLIENT_CERT_KEY || '.ssl/client.key',
+  connectionString: process.env.KAFKA_URL ? process.env.KAFKA_URL.replace(/\+ssl/g, '') : '',
+};
+
+const testConfig = {
+  topic: 'test-topic',
+  sslCert: 'test-cert',
+  sslKey: 'test-key',
+  connectionString: 'test-url',
+};
+
+const config = {
+  test: testConfig,
+  integration: herokuConfig,
+  production: herokuConfig,
+  staging: herokuConfig,
+};
+
+module.exports = {
+  getConfig: (environmentName) => {
+    if (!environmentName) environmentName = process.env.NODE_ENV;
+    return config[environmentName] ? config[environmentName] : config.test;
+  },
+
+  kafkaLogging: process.env.KAFKA_LOGGING,
+
+  // Local logging will only be off if process.env.LOCAL_LOGGING is false, default true
+  localLogging: process.env.LOCAL_LOGGING ? !(process.env.LOCAL_LOGGING === 'false') : true,
```

Comment:
For boolean configs, I personally prefer feature-toggles which we use in refocus as well (https://github.com/salesforce/refocus/blob/master/config/toggles.js). You can keep this in your books and use it wherever it makes sense. :)